src/react/components/shell/helpMenu.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import MessageBox from "../common/messageBox/messageBox";
+import { strings } from "../../../common/strings";
+import { KeyboardContext, IKeyboardContext, KeyEventType } from "../common/keyboardManager/keyboardManager";
+import { IKeyboardBindingProps, KeyboardBinding } from "../common/keyboardBinding/keyboardBinding";
+import "./helpMenu.scss";
+
+export interface IHelpMenuProps {
+    onClose?: () => void;
+}
+
+export interface IHelpMenuState {
+    show: boolean;
+}
+
+export class HelpMenu extends React.Component<IHelpMenuProps, IHelpMenuState> {
+    public static contextType = KeyboardContext;
+    public context!: IKeyboardContext;
+
+    public state = {
+        show: false,
+    };
+    private icon: string = "fa-question-circle";
+
+    public render() {
+        return (
+            <div className={"help-menu-button"} onClick={() => this.setState({show: true})}>
+                <i className={`fas ${this.icon}`}/>
+                <KeyboardBinding
+                    displayName={strings.editorPage.help.title}
+                    accelerators={["Ctrl+H", "Ctrl+h"]}
+                    handler={() => this.setState({show: !this.state.show})}
+                    icon={this.icon}
+                    keyEventType={KeyEventType.KeyDown}
+                />
+                <MessageBox
+                    title={strings.titleBar.help}
+                    message={this.getHelpBody()}
+                    show={this.state.show}
+                    onCancel={this.onClose}
+                    hideFooter={true}
```

Comment:
what's the difference between `show` and `hideFooter`?

Reply:
Show is to show the modal itself. HideFooter hides the footer in the modal. I'll udpate the docs